### PR TITLE
8.3 update to 1.1.1k-12

### PR DESCRIPTION
**Pull request by bleader**
- Sync with Centos 8 Stream's 1.1.1k-12
- do not package openssl.cnf file, to avoid conflict with base system openssl
- disable weak ciphers at compile time

### CODE REVIEW

File: SPECS/xs-openssl.spec [hunk 1]
```diff
@@ -539,6 +552,25 @@ export LD_LIBRARY_PATH
...
+￼-   Resolves: RHEL-17696
+￼- * Wed Nov 15 2023 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-11
+￼- - Fix CVE-2023-5678: Generating excessively long X9.42 DH keys or checking
+￼-   excessively long X9.42 DH keys or parameters may be very slow
+￼-   Resolves: RHEL-16538
+￼- * Thu Oct 19 2023 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-10
+￼- - Fix CVE-2023-3446: Excessive time spent checking DH keys and parameters
+￼-   Resolves: RHEL-14245
+￼- - Fix CVE-2023-3817: Excessive time spent checking DH q parameter value
+￼-   Resolves: RHEL-14239
```
Comment (stormi) [inline SPECS/xs-openssl.spec]: There's an invisible char (seen by github, though) at the start of each of these lines.

**Review by gthvn1 — APPROVED** [review]

File: SPECS/xs-openssl.spec (hunk 1 above)
Comment (bleader) [reply SPECS/xs-openssl.spec]: Indeed, that's in the merge commit, I'm trying to see if I can find a way to rebase including the merge commit to edit it, but for now, without luck.

Comment (bleader) [reply SPECS/xs-openssl.spec]: Don't take into account the last push force where I actually fixed it in last commit, I'll replace all that.

Comment (bleader) [reply SPECS/xs-openssl.spec]: Should be good: remove those, and reused the same changelog entry for the .cnf file as in 8.2.

**Review by stormi — APPROVED** [review]

**Comment by bleader** [comment]
Koji build (v8.3-incoming): http://koji.xcp-ng.org/taskinfo?taskID=59074

**Pull request merged**